apps/meteor/packages/rocketchat-i18n/i18n/de.i18n.json

```diff
@@ -715,7 +715,7 @@
   "Better": "Besser",
   "Bio": "Biografie",
   "Bio_Placeholder": "Biografie Platzhalter",
-  "Block": "Sperren",
+  "Block": "blockieren",
```

Comment:
`question` Capitalization is right?

Reply:
Let me double-check in similar Deutsch websites/systems ...

Reply:
Fixed